graylog2-rest-client/src/main/java/org/graylog2/restclient/models/Extractor.java

```diff
@@ -37,6 +38,7 @@
 
     public interface Factory {
         Extractor fromResponse(ExtractorSummaryResponse esr);
+        Extractor fromResponse(ExtractorSummary es);
```

Comment:
Should the old `fromResponse(ExtractorSummaryResponse)` method be marked as deprecated now?